src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexBoyerMoore.cs

```diff
@@ -43,15 +43,7 @@ public RegexBoyerMoore(string pattern, bool caseInsensitive, bool rightToLeft, C
             if (caseInsensitive)
             {
                 pattern = string.Create(pattern.Length, (pattern, culture), (span, state) =>
-                {
-                    // We do the ToLower character by character for consistency.  With surrogate chars, doing
```

Comment:
Is the issue described in this comment no longer relevant?

Reply:
Do we have a test for it?

Reply:
I don't think the comment ever applied, I don't really get the part with the consistency and I prefer to operate on the whole string not just because of convenience but also because of integrity, like the surrogate pairs that the author mentioned here.

Reply:
Yes we have several tests that hit these code paths. Our test bed is fortunately very rich.